pyreisejl/utility/extract_data.py

```diff
@@ -250,6 +256,12 @@ def _get_outputs_from_converted(matfile):
     except AttributeError:
         pass
 
+    try:
```

Comment:
Where do we expect an AttributeError to come from? For storage and DC lines, we expect it from a missing field in `mpc` (if no storage or no DC lines), but `mpc` will always have `bus`.

Reply:
Great point. When I was first figuring out how demand flexibility would be included, I anticipated having a set that identified demand flexibility nodes (similar to storage). However, I saw what was done with the `bus` field and ended up going with that instead. Even once feature reduction is implemented, I can still probably set the output IDs using the `bus` field, so something like this for demand flexibility won't likely be needed in the future either. I will remove this.

Reply:
Done.